float32 goal_position
float32 goal_velocity
float32 goal_acceleration
float32 goal_current
---
bool goal_accepted
